### ISSUE TITLE
Variant-Phenotype distant supervision examples are missing the gene/isoform name

### ISSUE THREAD

**Issue by amwenger** [ajratner/dd-genomics#109]
The "c." and "p." HGVS notations are gene-based and can only be understood in reference to a specific gene isoform.  For example, "SRF:c.25A>G" describes a variant of basepair 25 of the SRF gene (reference is A, alternative is G).  The "c.25A>G" has no frame of reference without the gene name.  In fact, because many genes have multiple isoforms, the gene name is not enough to resolve ambiguity.  It is best to use an isoform identifier (you'll see "NM_######" RefSeq isoform identifiers in ClinVar) to be unambiguous.

The `hgvs_to_hpo.tsv` distant supervision dataset produced by `make_all.sh` is missing a gene/isoform name to provide a frame of reference.  Example lines are:

> c.289C>T|p.Arg97Cys   HP:0000007|HP:0003159|HP:0008672
> c.1786C>T|p.Arg596Ter   HP:0000007|HP:0000510|HP:0000543|HP:0000662|HP:0001133|HP:0007737

**Comment by amwenger** [comment]
Perhaps this is a duplicate of #107 

The gene name is available in the ClinVar download file.

**Comment by gbgbg** [comment]
Possible aside:
While V-P relationships are no good without a clear V-G relation as Aaron noted, pairwise connections of the type G-V are of mild interest even without P, as they suggest a specific variant was studied in more details by someone. (And as there 3 billion base pairs its non trivial that someone actually studied a patient mutation that may be otherwise of interest to us). 
@amwenger: best ignore these for now / forever?

-Gill

**Comment by amwenger** [comment]
I agree with @gbgbg.  It is good to be aware of every variant mention, even if it is not connected to a phenotype.  I would extract them even now.
